flink-examples/flink-examples-streaming/src/main/java/org/apache/flink/streaming/examples/windowing/TopSpeedWindowing.java

```diff
@@ -55,6 +55,7 @@ public static void main(String[] args) throws Exception {
 		final StreamExecutionEnvironment env = StreamExecutionEnvironment.getExecutionEnvironment();
 		env.setStreamTimeCharacteristic(TimeCharacteristic.EventTime);
 		env.getConfig().setGlobalJobParameters(params);
+		env.setParallelism(1);
```

Comment:
Is this strictly needed? It's not a Unit or ITCase and the example seems to work without this line.

Reply:
Yes it's because of `TopSpeedWindowingExampleITCase`. However, it's not very nice and I decided to rework the `TopSpeedWindowingExampleITCase` instead.